app/views/user_mailer/email_confirmation_instructions.html.erb

```diff
@@ -0,0 +1,38 @@
+<p class="lead">
+  <%= t('user_mailer.email_confirmation_instructions.header',
+        intro: @first_sentence) + ' ' + t('user_mailer.email_confirmation_instructions.footer',
```

Comment:
Adding the space in manually is confusing to me...let's just split this up into two different things? HTML simplifies extra whitespace so a newline will look the same

```erb
<p class="lead">
  <%= t('user_mailer.email_confirmation_instructions.header', intro: @first_sentence) %>
  <%= t('user_mailer.email_confirmation_instructions.footer', confirmation_period: @confirmation_period) %>
</p>
```

Reply:
Good point - this code was in the slim template, but I should have cleaned it up during conversion to ERB.